consensus/quorum/one-node-staked-vote.go

```diff
@@ -97,15 +97,16 @@ func (v *stakedVoteWeight) computeCurrentTotalPower(p Phase) (*numeric.Dec, erro
 	}()
 
 	for i := range members {
-		w.FromLibBLSPublicKey(members[i])
+		if err := w.FromLibBLSPublicKey(members[i]); err != nil {
+			return nil, err
+		}
 		if _, didVote := ballot.voters[w]; !didVote &&
 			v.ReadBallot(p, members[i]) != nil {
-			err := w.FromLibBLSPublicKey(members[i])
-			if err != nil {
+			if err := w.FromLibBLSPublicKey(members[i]); err != nil {
```

Comment:
existing problem, but duplicated "w.FromLibBLSPublicKey"

Reply:
Only moved line to scope the error